config/main.py

```diff
@@ -5429,6 +5429,83 @@ def disable_use_link_local_only(ctx, interface_name):
     interface_dict = db.get_table(interface_type)
     set_ipv6_link_local_only_on_interface(db, interface_dict, interface_type, interface_name, "disable")
 
+#
+# 'tx_error_threshold' subgroup ('config tx_error_threshold <interface_name> ...')
+#
+
+@interface.group('tx_error_threshold')
+@click.pass_context
+def tx_error_threshold(ctx):
+    """Set or del threshold of tx error statistics"""
+    pass
+
+#
+# 'set_thresh' subcommand
+#
+
+@tx_error_threshold.command()
+@click.argument('interface_name', metavar='<interface_name>', required=True)
+@click.argument('interface_tx_err_threshold', metavar='<interface_tx_err_threshold>', required=True, type=int)
+@click.pass_context
+def set_thresh(ctx, interface_name, interface_tx_err_threshold):
+    config_db = ctx.obj['config_db']
+
+    """Set threshold of tx error statistics"""
+    if interface_name is None:
```

Comment:
`if not interface_name` in Python will work in all cases regardless of the type of the variable. Please change it